auditbeat/docs/fields.asciidoc

```diff
@@ -6316,6 +6316,16 @@ type: object
 Kubernetes annotations map
 
 
+--
+
+*`kubernetes.replicaset`*::
```

Comment:
Is this the PR where you saw the errors you posted into the other issue?

Reply:
Yes